vllm/model_executor/models/glm4_vision_encoder.py

```diff
@@ -79,6 +80,31 @@ def __init__(
         self.output_dropout = torch.nn.Dropout(config.dropout_prob)
 
     def forward(self, x: torch.Tensor) -> torch.Tensor:
+        _ON_NAVI3 = "gfx11" in torch.cuda.get_device_properties("cuda").gcnArchName
+        if _ON_NAVI3:
+            try:
+                # git clone -b howiejay/navi_support https://github.com/ROCm/flash-attention.git
```

Comment:
Will this fail without flash_attn built from this particular branch?

Reply:
yes, only this branch support navi3.

Reply:
Will any other FA branch trigger a ModuleNotFoundError?
Is there an ETA on when this branch is going to be accepted into mainline FA?

Reply:
The other FA branch doesn't support navi3. It will be fail to install on gfx1100. I think there isn't plan to upstream this branch.